include/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+set(SOURCE_FILES 
+    tack_dummy.cpp
```

Comment:
Also weird, I think its possible to have header only "libraries" in CMake 3.x like this.

```cmake
add_library(tack INTERFACE)
target_include_directories(tack INTERFACE ${CMAKE_CURRENT_SOURCE_DIR})
```

This will not produce any compiled objects but allows CMake to get the include directory by "linking" against this dummy target, the `target_include_directories`  is to specify the path to the include directories.